src/OWML.ModHelper.Interaction/ModInteraction.cs

```diff
@@ -74,15 +74,12 @@ public IList<IModBehaviour> GetDependencies(string uniqueName)
 		public TInterface GetModApi<TInterface>(string uniqueName) where TInterface : class
 		{
 			var api = GetApi(uniqueName);
-			switch (api)
+			return api switch
 			{
-				case null:
-					return null;
-				case TInterface inter:
-					return inter;
-				default:
-					return _proxyFactory.CreateProxy<TInterface>(api, _manifest.UniqueName, uniqueName);
-			}
+				null => null,
+				TInterface inter => inter,
+				_ => _proxyFactory.CreateProxy<TInterface>(api, _manifest.UniqueName, uniqueName)
+			};
```

Comment:
yeah, just not a fan of how these new return switches look :/

Reply:
I always hated the old switch syntax and fell in love with switch expressions right away. They're the main reason I upgraded the projects and C# versions :P